build/config/coverage/BUILD.gn

```diff
@@ -7,7 +7,7 @@ import("//build/config/coverage/coverage.gni")
 config("default_coverage") {
   if (use_clang_coverage) {
     ldflags = []
-    if (!is_win) {
+    if (!is_win && !sb_is_evergreen) {
```

Comment:
Customizations to 3p libraries should be wrapped with `use_cobalt_customizations`

Reply:
I think I can remove this line entirely. Iirc I added it when I was trying to get coverage to work on all platforms. I'll deal with this when I'm actually going to add coverage for evergreen runs.